src/App.js

```diff
@@ -3,7 +3,7 @@ import './App.css';
 
 //Holds one true value
 let valueBoard = [];
-let mode = "easy";
+let difficultyValue = 46;
```

Comment:
`numbersToHide` or `numbersToRemove` is probably better.

Reply:
Done